lib/list/ensureTextIndex.js

```diff
@@ -89,7 +89,9 @@ function ensureTextIndex (callback) {
 			}
 
 			// It's a text index but not one of ours; nothing we can do
-			debug('Existing text index \'' + existingIndexName + '\' in \'' + list.key + '\' not recognised; no action taken');
+			console.error('list.ensureTextIndex() failed to update the existing text index \'' + existingIndexName + '\' for the \'' + list.key + '\' list.');
```

Comment:
These comments would be better IMO as one line with `\n` line break characters, keeps them grouped in log aggregation environments as a single piece of information, not three. I'll merge & clean that up.